### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by kmaric69**

### CODE REVIEW

File: src/ThemeOptions/ThemeOptionsExample.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function createThemeOptionsPage(): void
 	 */
 	public function registerThemeOptions(): void
 	{
-		if (function_exists('acf_add_options_page')) {
+		if (function_exists('acf_add_local_field_group')) {
```
Comment (gabriel-glo) [inline src/ThemeOptions/ThemeOptionsExample.php:76]: Nice

**Review by iruzevic — APPROVED** [review]

**Review by iobrado — APPROVED** [review]

**Review by dingo-d — CHANGES_REQUESTED** [review]
I'm not on the list and I disapprove of this! 

:trollface:

**Comment by kmaric69** [comment]
I will close this PR because @dingo-d doesn't feel included.

**Comment by dingo-d** [comment]
Rightly so! I approve of the closing due to the lack of inclusion! 😄

**Pull request closed without merging**